<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaintWindow</class>
 <widget class="QMainWindow" name="PaintWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox_BrushSize">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>50</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>2px</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4px</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8px</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12px</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16px</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32px</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_BrushColor">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>White</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Black</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dark Gray</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Gray</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Light Gray</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cyan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Magenta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Yellow</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dark Red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dark Green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dark Blue</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dark Cyan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dark Magenta</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dark Yellow</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_BrushSize">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>20</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Brush Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_BrushColor">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Brush Color</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Instruments">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>160</y>
      <width>125</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Instruments</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>200</y>
      <width>131</width>
      <height>301</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="BrushButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Brush</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/paint-brush.png</normaloff>:/icons/paint-brush.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PenButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Pencil</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/pencil.png</normaloff>:/icons/pencil.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EraserButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Eraser</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/eraser.png</normaloff>:/icons/eraser.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DropperButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Dropper</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/pipette.png</normaloff>:/icons/pipette.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LineButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Line</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/line.png</normaloff>:/icons/line.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RectangleButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Rectangle</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/rectangle.png</normaloff>:/icons/rectangle.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CircleButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Circle</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/circle.png</normaloff>:/icons/circle.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ClearButton">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Clear Canvas</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/clear-mop.png</normaloff>:/icons/clear-mop.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_CurentColor">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>510</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuSave">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveImageAs"/>
    <addaction name="actionSave_to_my_files"/>
    <addaction name="actionOpen_from_my_files"/>
   </widget>
   <addaction name="menuSave"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSaveImageAs">
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionSave_to_my_files">
   <property name="text">
    <string>Save to your Folder</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionOpen_from_my_files">
   <property name="text">
    <string>Open from your files</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit App</string>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
